release-notes/3.0/preview/api-diff/preview5/standalone-packages/3.0-preview5-standalone-packages.md

```diff
@@ -0,0 +1,7 @@
+# API Difference netcoreapp3.0-preview4 vs netcoreapp3.0-preview5
```

Comment:
nit: I would change the title since this isn't about netcoreapp3.0 anymore.

Maybe something like:
`API Difference standalone-preview4 vs standalone-preview5.`